alica_engine/src/engine/modelmanagement/factories/KeyMappingFactory.cpp

```diff
@@ -12,8 +12,14 @@ std::unique_ptr<KeyMapping> KeyMappingFactory::create(const YAML::Node& node)
     if (Factory::isValid(node[alica::Strings::input])) {
         const auto& inputList = node[alica::Strings::input];
         for (const auto& entry : inputList) {
-            keyMapping->addInputMapping(
-                    Factory::getValue<std::string>(entry, alica::Strings::parentKey), Factory::getValue<std::string>(entry, alica::Strings::childKey));
+            // TODO: Remove ifelse once backend has been updated and all keymapping nodes have an (empty) value field
```

Comment:
I don't follow this comment. The value field need not always be empty right? Either parent key or value needs to be filled